docs/telegram-usage.md

```diff
@@ -262,14 +266,12 @@ Below, example of Telegram message you will receive for each command.
 
 ### /stopbuy
 
-> **status:** `Setting max_open_trades to 0. Run /reload_config to reset.`
-
-Prevents the bot from opening new trades by temporarily setting "max_open_trades" to 0. Open trades will be handled via their regular rules (ROI / Sell-signal, stoploss, ...).
+Alias of the `/pause` command.
```

Comment:
I'd even go so far to remove the `/stopbuy` section here completely, and name the upper one `/pause | /stopbuy` - making it crystal clear that it's an alias.
there's little point in having 2 sections - which "kinda" try to explain the same - but do so in quite different wording.